src/test/java/seedu/address/logic/commands/DeletePrescriptionCommandTest.java

```diff
@@ -0,0 +1,129 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandPrescriptionTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandPrescriptionTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.CommandPrescriptionTestUtil.showPrescriptionAtIndex;
+import static seedu.address.logic.commands.DeletePrescriptionCommand.MESSAGE_DELETE_PRESCRIPTION_SUCCESS;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PRESCRIPTION;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PRESCRIPTION;
+import static seedu.address.testutil.TypicalPrescriptions.getTypicalPrescriptionList;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code DeletePrescriptionCommand}.
+ */
+public class DeletePrescriptionCommandTest {
+    private seedu.address.model.ModelPrescription model;
+    private seedu.address.model.ModelPrescription expectedModel;
+
+    @BeforeEach
+    public void setUp() {
+        model = new seedu.address.model.ModelManagerPrescription(getTypicalPrescriptionList(),
+                new seedu.address.model.UserPrefsPrescription());
+        expectedModel = new seedu.address.model.ModelManagerPrescription(model.getPrescriptionList(),
+                new seedu.address.model.UserPrefsPrescription());
+    }
+    @Test
+    public void execute_validIndexUnfilteredList_success() {
+        seedu.address.model.prescription.Prescription prescriptionToDelete = model.getFilteredPrescriptionList()
+                .get(INDEX_FIRST_PRESCRIPTION.getZeroBased());
+        DeletePrescriptionCommand deletePrescriptionCommand = new DeletePrescriptionCommand(INDEX_FIRST_PERSON);
+
```

Comment:
Inconsistent imports.

Should remove all instances of `INDEX_XXX_PERSON` and change to `INDEX_XXX_PRESCRIPTION` as the Person variant will be deprecated.